### DISCUSSION TITLE
How to Programmatically Publish/Republish a Job using API Endpoints

### DISCUSSION THREAD

**Discussion by mikerubin** [bullhorn/career-portal#522] [Q&A]
Does anyone know how to programmatically (re)published an open Bullhorn job (isOpen=true) either via an HTTP POST request to the JobOrder entity or via some other method?
I don't see anything in the Bullhorn documentation and all my Google searches have drawn blanks.
I opened a help desk ticket with Bullhorn and they confirmed jobs can be published/republished via an API, but they directed me to the bullhorn / career-portal for further assistance.
Thank you in advance.
Kind regards.
Mike.

**Comment by pbearne** [comment]
With full private API, you can set isPublished to true
you might want to update the date etc.

you need to be careful here, as Google may ding you 
What are you trying to do?

Paul
Matadorjobs.com
The WordPress Job Board for Bullhorn

**Comment by mikerubin** [comment]
Hi Paul.
Thanks for your reply.
Nothing nefarious. We hire for the same position many times over with
little or no change to the job description. We're writing a stand-along
utility outside of Bullhorn to streamline the process.
I don't see isPublished in the JobOrder entity. Am I looking in the wrong
place?

**Reply by pbearne** [reply]
My bad, it is called published. But the id is isPublic

Assuming you are using the OSCP or job board that follows the pattern. 

Paul

**Comment by mikerubin** [comment]
Thanks. I'll give it a shot and let you know how I make out.
Mike.
